Makefile

```diff
@@ -55,6 +55,7 @@ postsubmit-build:
 
 .PHONY: postsubmit-conformance
 postsubmit-conformance: postsubmit-build
+	bash development/kops/prow.sh
```

Comment:
Do we need to call bash here or could we call the script directly like elsewhere in this file?